### ISSUE TITLE
Add local charger without cloud part?

### ISSUE THREAD

**Issue by tobiasehlert** [ricott/homey-no.easee#4]
Hi @ricott,

Is there a possibility to add a Easee charger without requiring to login to the cloud?
I got one at home that is connected to Tibber and I can't add it to Homey, since the Homey Easee charger app says it can't find a charger on my account in the cloud.

Regards,
Tobias

**Comment by ricott** [comment]
Hi Tobias!

To my knowledge, there is no local API for Easee (yet). 
You can upvote on ideas here, https://easee.upvoty.com/b/ideas/.

Not sure why you wouldn't be able to add your charger to Homey. Can you see your charger if you login to https://easee.cloud/? You should login to Homey using same phone number and password.

**Comment by tobiasehlert** [comment]
Hi @ricott,

I had a "site" created on the cloud portal since earlier and added the charger there. With that setup, it was not possible to add the charger to Homey. I also added the charger under "my products" and after that, it was possible to add it in the Homey app.
So now I got it on two places on easee.cloud, but it doesn't really matter, since I'm not in there anyways.

Thanks for your plugin, really great (from what I've tested for 6 minutes now) 👍 
Closing the issue, since it's resolved now.

Regards,
Tobias
